### PR TITLE
Frontend/order completion

### PR DESCRIPTION
**Pull request by graham-willis-druid**
1036 - CSRF protection
1037 - Add CSP
1038 - Add robots (provisional file) - changed as suggested, built by gulp
1039 - Add the back-button - no Javascript
Fixed error logging issue
Added order-completion page and PDF downloads
Added guidance for refunds, cookies, privacy and a placeholder for the accessibility statement
Terminate the service if started twice
Added styles to support printout and order confirmation to sass

### CODE REVIEW

File: packages/gafl-webapp-service/src/gafl-webapp-service.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
 /**
- * Start the hapi
+ * Create and start the web-server (once)
  */
-import { createServer, init } from './server.js'
 createServer({ port: 3000 })
-init()
+init().catch(e => {
+  console.error(e)
+  process.exit()
```
Comment (sgardnerdell-ea) [inline packages/gafl-webapp-service/src/gafl-webapp-service.js:9]: Good idea to use a non-zero exit code here to indicate a failure

File: packages/gafl-webapp-service/src/pages/order-complete/pdf-route.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    const doc = printer.createPdfKitDocument(pdfDoc)
+    const chunks = []
+
+    doc.on('data', function (chunk) {
+      chunks.push(chunk)
+    })
+    doc.on('end', function () {
+      resolve(Buffer.concat(chunks))
+    })
+    doc.end()
```
Comment (sgardnerdell-ea) [inline packages/gafl-webapp-service/src/pages/order-complete/pdf-route.js:28]: Should it be listening for the error event here and rejecting the promise?  As in:
```
doc.on('error', (err) => {
  reject(err);
});
```

Comment (graham-willis-druid) [reply packages/gafl-webapp-service/src/pages/order-complete/pdf-route.js:28]: Error is caught later by the server exception handler - this is harder to mock and is not in the example code provided.

File: packages/gafl-webapp-service/src/gafl-webapp-service.js (hunk 1 above)
Comment (graham-willis-druid) [reply packages/gafl-webapp-service/src/gafl-webapp-service.js:9]: Fixed

File: packages/gafl-webapp-service/src/pages/order-complete/order-complete.njk [hunk 3]
```diff
@@ -3,25 +3,142 @@
...
+{% set bailiffMessage %}
+<p class="govuk-body">If our bailiffs ask to see your licence, you must show them:</p>
+    <ol class="class="govuk-list govuk-list--number">
+        <li class="govuk-body"><span class="govuk-!-font-weight-bold">Proof of your licence: </span><a class="no-print" href="javascript:void(0);" onclick="printDocument();">print out this page</a><span class="print-area">take this printout with you</span> or make a note of your licence number</li>
+        <li class="govuk-body"><span class="govuk-!-font-weight-bold">Proof of your identity: </span>such as your driver's licence or other photo ID</li>
+    </ol>
+{% endset -%}
+
+{% if data.permission.licenceLength === '1D' %}
+    {% set lengthText = '1 Day' %}
```
[the remark this answers is not in the document]
Comment (graham-willis-druid) [reply packages/gafl-webapp-service/src/pages/order-complete/order-complete.njk:20]: Fixed throughout

File: packages/gafl-webapp-service/src/pages/guidance/cookies.njk [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        <h2 class="govuk-heading-m">Session and security cookies</h2>
+
+        <p class="govuk-body">We store session cookies on your computer to help keep your information secure while you use the service.</p>
+
+        {{ govukTable({
+          caption: "Session cookie details",
+          firstCellIsHeader: true,
+          head: [ { text: "Name" }, { text: "Purpose" }, { text: "Expires" } ],
+          rows: [
+            [ { text: "sid" }, { text: "Used for session management" }, { text: "3 hours after you stop using the service" } ],
```
[the remark this answers is not in the document]
Comment (graham-willis-druid) [reply packages/gafl-webapp-service/src/pages/guidance/cookies.njk:37]: Done

**Pull request merged**